packages/patternfly-4/react-core/src/internal/util.js

```diff
@@ -1,3 +1,12 @@
 export function capitalize(input) {
   return input[0].toUpperCase() + input.substring(1);
 }
+
+export function getUniqueId() {
```

Comment:
might be nice to allow a prefix for this so that we can make the id's somewhat useful
```js
getUniqueId(prefix = '') {
  return (
   `${prefix}-` + new Date().getTime() +
    Math.random()
      .toString(36)
      .slice(2)
  );
}
```
this would allow us to do something like `getUniqueId('pf-switch')` we could even add the `pf-` by default and have the preson just pass the component name to it.